fixes #3940: document changes to several config options (#3986)
webpack/webpack.js.org: src/content/configuration/output.md

  //...
  output: {
    //...
    chunkLoadingGlobal: 'myCustomFunc'
  }
};
```

## `output.chunkLoading`

> `boolean = false` `string: 'jsonp' | 'import-scripts' | 'require' | 'async-node' | <any string>`

Reviewer: ```suggestion
`false` `string: 'jsonp' | 'import-scripts' | 'require' | 'async-node' | <any string>`
```

> The method of loading chunks (methods included by default are 'jsonp' (web), 'importScripts' (WebWorker), 'require' (sync node.js), 'async-node' (async node.js), but others might be added by plugins).

Reviewer 2: ```suggestion
The method to load chunks (methods included by default are 'jsonp' (web), 'importScripts' (WebWorker), 'require' (sync node.js), 'async-node' (async node.js), but others might be added by plugins).
```

__webpack.config.js__

```javascript
module.exports = {
  //...
  output: {
    //...
    chunkLoading: 'async-node'
  }
};
```

## `output.chunkFormat`

> `boolean = false` `string: 'array-push' | 'commonjs' | <any string>`

Reviewer: ```suggestion
`false` `string: 'array-push' | 'commonjs' | <any string>`
```

The format of chunks (formats included by default are 'array-push' (web/WebWorker), 'commonjs' (node.js), but others might be added by plugins).

__webpack.config.js__

```javascript
module.exports = {
  //...
  output: {
    //...
    chunkFormat: 'commonjs'
  }
};
```

## `output.enabledChunkLoadingTypes`

> `[string]` `string: 'jsonp' | 'import-scripts' | 'require' | 'async-node' | <any string>`

Reviewer: ```suggestion
`[string: 'jsonp' | 'import-scripts' | 'require' | 'async-node' | <any string>]`
```

`string = 'window'`

> When targeting a library, especially when `libraryTarget` is `'umd'`, this option indicates what global object will be used to mount the library. To make UMD build available on both browsers and Node.js, set `output.globalObject` option to `'this'`. Defaults to `self` for web-like targets.

Reviewer 2: ```suggestion
When targeting a library, especially when `libraryTarget` is `'umd'`, this option indicates what global object will be used to mount the library. To make UMD build available on both browsers and Node.js, set `output.globalObject` option to `'this'`. Defaults to `self` for Web-like targets.
```

What are the Web like targets?

`output.uniqueName` will be used to generate unique globals for:

- [`output.chunkLoadingGlobal`](/configuration/output/#outputChunkLoadingGlobal)

> - [`output.chunkLoadingGlobal`](/configuration/output/#outputChunkLoadingGlobal)

Reviewer 2: Please remove one

Reviewer: ```suggestion
```

`output.uniqueName` will be used to generate unique globals for:

> - [`output.chunkLoadingGlobal`](/configuration/output/#outputChunkLoadingGlobal)

Reviewer: ```suggestion
- [`output.chunkLoadingGlobal`](/configuration/output/#outputchunkloadingglobal)
```

T> Typically you don't need to change `output.hotUpdateChunkFilename`.

## `output.hotUpdateGlobal`

`string`

Only used when [`target`](/configuration/target/) is set to `'web'`, which uses JSONP for loading hot updates.

A JSONP function used to asynchronously load hot-update chunks.

> For details see [`output.chunkLoadingGlobal`](#outputChunkLoadingGlobal).

Reviewer: ```suggestion
For details see [`output.chunkLoadingGlobal`](#outputchunkloadingglobal).
```
